contributing/code/standards.rst

```diff
@@ -51,6 +51,18 @@ example containing most features described below:
         }
 
         /**
+         * @return string
+         *
+         * @deprecated
+         */
+        public function someDeprecatedMethod()
+        {
+            @trigger_error(sprintf('The %s() method is deprecated since version 2.8 and will be removed in 3.0. Use Acme\Baz::someMethod() instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
It's mixed. Sometimes we use it, sometimes we don't.

@symfony/deciders What do you think? Should we advise to use `sprintf()`?